hooks/post_gen_project.py

```
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+import os
+import platform
+import subprocess
+import shutil
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)